packages/web-components/fast-components/src/styles/patterns/button.ts

```diff
@@ -90,7 +91,9 @@ export const BaseButtonStyles = (
             font: inherit;
         }
         .control.icon-only {
-            padding: 0;
+            ${densityComponentHorizontalOuterPadding.cssCustomProperty}: var(${
```

Comment:
One option for making something square. This assumes the vertical measurement should stay consistent, which _implies_ that we think the horizontal measurement probably is different. This works with most buttons I've seen, but may not be a universal application.

Related to the previous comment about not being able to set token values for a particular selector, the more semantically correct alternative would be to set the horizontal `Units` to the same value as vertical and let the `Padding` be calculated.